services/worker/src/worker/job_runners/config/split_names_from_streaming.py

```diff
@@ -109,7 +100,7 @@ def compute_split_names_from_streaming_response(
             f"Cannot get the split names for the config '{config}' of the dataset.",
             cause=err,
         ) from err
-    return {"splits": split_name_items}
+    return SplitsList({"splits": split_name_items})
```

Comment:
I didn't know we could instantiate a "TypedDict" class. I was using them only as type hints. It works well, interesting!